.github/labels.yml

```
@@ -15,6 +15,22 @@ rules:
   - labels: ['dependencies']
     patterns: ['requirements/**|package.json|yarn.lock']
 
+  - labels: ['Expected Risk: High']
```

can we drop the word `Expected` and just make it `Risk:`?
```suggestion
  - labels: ['Risk: High']
```

---

sure - 083e0272d66